### PR TITLE
docs: add `snapshot_cron` flag in README (English and zh-CN)

### PR DESCRIPTION
**Pull request by Niennienzz**
- Add docs for the `snapshot_cron` flag in `README.md` and `README.zh-CN.md`.
- A few formatting minor changes in `README.zh-CN.md`.
- There's one place `Memcached` is mis-spelled as `Memcache`. Fixed in all languages.

### CODE REVIEW

**Comment by romange** [comment]
@Niennienzz  there are some whitespace problems

**Comment by Niennienzz** [comment]
> @Niennienzz there are some whitespace problems

Thank you @romange, fixed!

I think the only CI failing is because of timeout?

File: README.md [hunk 1]
```diff
@@ -106,7 +106,8 @@ There are also some Dragonfly-specific arguments:
  * `dbnum`: Maximum number of supported databases for `select`.
  * `cache_mode`: See the [novel cache design](#novel-cache-design) section below.
  * `hz`: Key expiry evaluation frequency (`default: 100`). Lower frequency uses less CPU when idle at the expense of a slower eviction rate.
- * `save_schedule`: Glob spec for the UTC to save a snapshot in HH:MM (24h time) format (`default: ""`).
+ * `snapshot_cron`: Cron schedule expression for automatic backup snapshots using standard cron syntax with the granularity of minutes (`default: ""`).
```
Comment (romange) [inline README.md:109]: it's not straightforward so I think it's a great place to add some examples with the explanation.

Comment (Niennienzz) [reply README.md:109]: Added examples, linked to our "Saving Backups" doc. 😃

File: README.md [hunk 2]
```diff
@@ -106,7 +106,21 @@ There are also some Dragonfly-specific arguments:
...
+   Here are some cron schedule expression examples below, and feel free to read more about this argument in our [documentation](https://www.dragonflydb.io/docs/managing-dragonfly/backups#the-snapshot_cron-flag).
+
+   | Cron Schedule Expression	 | Description                                    |
+   |---------------------------|------------------------------------------------|
+   | `* * * * *`               | At every minute                                |
+   | `*/5 * * * *`             | At every 5th minute                            |
+   | `*/30 * * * *`            | At every 30th minute                           |
```
Comment (romange) [inline README.md:116]: Thanks, let's remove lines 116, 120, 117 :)

**Review by romange — APPROVED** [review]

**Pull request merged**